CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Provide `User` without `Option` [#70](https://github.com/maxcountryman/axum-login/pull/70)
 - Use associated type `Error` in `UserStore` instead of eyre for error handling [#69](https://github.com/maxcountryman/axum-login/pull/69)
 - Make `role_bounds` optional [#67](https://github.com/maxcountryman/axum-login/pull/67)
+- Update `sqlx` to 0.7, which [drops support for MS SQL Server](https://github.com/launchbadge/sqlx/pull/2039)

```

Comment:
Sorry this should go under the unreleased heading.

Reply:
ah oops sorry

Reply:
done